src/13_file_io.py

```diff
@@ -11,9 +11,19 @@
 
 # YOUR CODE HERE
 
+with open('src/foo.txt') as y:
+   print(y.read())
```

Comment:
Again, perfectly fine. Note that you can loop through `y` using a for-loop and print each line as well. You may see this on the next Sprint Challenge.